falcon/request.py

```diff
@@ -683,6 +687,22 @@ def access_route(self):
     def remote_addr(self):
         return self.env.get('REMOTE_ADDR')
 
+    @property
+    def port(self):
+        try:
+            host_header = self.env['HTTP_HOST']
+            host, port = parse_host(host_header)
+        except KeyError:
+            port = self.env.get('SERVER_PORT')
+        return port
+
+    @property
+    def netloc(self):
+        try:
+            return self.env['HTTP_HOST']
+        except KeyError:
+            return self.host + ':' + self.port
```

Comment:
Should this behave like [request.py#L560](https://github.com/falconry/falcon/blob/master/falcon/request.py#L560)?